src/components/requestResetPassword/EmailTextField.tsx

```diff
@@ -0,0 +1,46 @@
+import { FormHelperText, InputAdornment, Stack, TextField } from "@mui/material";
+import EmailIcon from "@mui/icons-material/Email";
+
+type props = {
+  placeholder: string;
+  value: string;
+  handleValueChange: (event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>) => void;
+  isErr: boolean;
+  errMsg: string;
+};
+
+export default function EmailTextField(props: props) {
+  return (
+    <>
+      <Stack spacing={1}>
+        <TextField
+          sx={{ backgroundColor: "#ffffff" }}
+          fullWidth
+          placeholder={props.placeholder}
+          value={props.value}
+          error={props.isErr}
+          onChange={props.handleValueChange}
+          inputProps={{
+            sx: {
+              textAlign: "center",
+              "&::placeholder": {
+                textAlign: "center",
+              },
+            },
+          }}
+          InputProps={{
+            startAdornment: (
+              <InputAdornment position="start">
+                <EmailIcon color={props.isErr ? "error" : "secondary"} />
+              </InputAdornment>
+            ),
+          }}
+        />
+        <FormHelperText error>
+          {props.isErr && props.errMsg}
+          {"\u00A0"}
```

Comment:
when there're only space in the field, the error message should be ช่องนี้ไม่สามารถเว้นว่างได้ , same as when text field is empty
<img width="1530" alt="Screenshot 2566-02-25 at 14 10 29" src="https://user-images.githubusercontent.com/62459518/221344229-e89c42b6-8c64-4c49-933f-0c9111664e69.png">

Reply:
Solved. Pls check.